specification/containerregistry/data-plane/Azure.ContainerRegistry/stable/2021-07-01/containerregistry.json

```diff
@@ -1177,6 +1186,55 @@
             "$ref": "./examples/PostAccessToken.json"
           }
         }
+      },
+      "get": {
+        "tags": [
+          "AcrToken",
+          "AccessToken",
+          "userLogin"
+        ],
+        "description": "Exchange Username, Password and Scope an ACR Access Token",
```

Comment:
missing a `for`? 

"description": "Exchange Username, Password and Scope **for** an ACR Access Token",